aws/basic-eks-cluster/cloudformation/eks-workers.yaml

```diff
@@ -0,0 +1,333 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
Is it possible to simplify this blueprint and have as few as possible cloudformation templates?

Reply:
This is the recommended EKS setup so let's stick to it, else we will have one huge CF file